frontend/Gruntfile.js

```diff
@@ -25,6 +21,17 @@ module.exports = function(grunt) {
         }
       }
     },
+    babel: {
+        options: {
+            sourceMap: true,
+            presets: ['es2015']
+        },
+        dist: {
+            files: {
+              '../pimpmycause/static/scripts/pimpmycause.min.js':'src/scripts/scripts.js',
```

Comment:
Created a ticket to improve the grunt file rather than block this PR https://womenhackfornonprofits.atlassian.net/browse/PIMP-316